data/test/vtgate/unsupported_cases.txt

```diff
@@ -291,33 +291,9 @@
 "delete from unsharded where col = (select id from unsharded join user on unsharded.id = user.id)"
 "unsupported: sharded subqueries in DML"
 
-# update with no where clause
-"update user set val = 1"
-"unsupported: multi-shard where clause in DML"
-
-# update with non-comparison expr
-"update user set val = 1 where id between 1 and 2"
-"unsupported: multi-shard where clause in DML"
-
-# update with primary id through IN clause
-"update user set val = 1 where id in (1, 2)"
-"unsupported: multi-shard where clause in DML"
-
-# update with non-unique key
-"update user set val = 1 where name = 'foo'"
-"unsupported: multi-shard where clause in DML"
-
-# update with no index match
-"update user set val = 1 where user_id = 1"
-"unsupported: multi-shard where clause in DML"
-
-# update by lookup with IN clause
-"update music set val = 1 where id in (1, 2)"
-"unsupported: multi-shard where clause in DML"
-
-# update with where clause with parens
-"update user set val = 1 where (name = 'foo' or id = 1)"
-"unsupported: multi-shard where clause in DML"
+# scatter update with limit clause
+"update user_extra set val = 1 where (name = 'foo' or id = 1) limit 1"
+"unsupported: multi shard update with limit"
```

Comment:
Please add an unuspported test case for the multi shard update with owned lookup vindex